analyses/cell-type-ewings/scripts/01-run-aucell.R

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env Rscript
+
+# This script is used to classify tumor and normal cells using AUCell with a marker gene list 
+# The threshold for defining cells as tumor cells should be determined separately by running 
+# `00-identify-ref-aucell.R`
+
+project_root <- here::here()
+renv::load(project_root)
+
+library(optparse)
+library(SingleCellExperiment)
+
+option_list <- list(
+  make_option(
+    opt_str = c("--sce_file"),
+    type = "character",
+    help = "Path to RDS file containing a processed SingleCellExperiment object from scpca-nf 
+      to be used to run AUCell."
+  ),
+  make_option(
+    opt_str = c("--marker_genes_file"),
+    type = "character",
+    default = file.path(project_root, "references", "tumor-marker-genes.tsv"),
+    help = "Path to file containing all marker genes used to run AUCell. 
+      Must contain `cell_type` and `ensembl_gene_id` columns.
+      Any marker genes where `cell_type` is 'Tumor' will be used."
+  ),
+  make_option(
+    opt_str = c("--auc_threshold"),
+    type = "double",
```

Comment:
Just noting that you are reading this in not from a file, but as a raw value. Is the assumption that you will use something like `--auc_threshold $(cat threshold_file.txt)` in a script to feed in the value?